src/java/voldemort/server/PosixHostMatcher.java

```diff
@@ -0,0 +1,19 @@
+package voldemort.server;
+
+import java.util.List;
+
+import voldemort.cluster.Node;
+
+
+public class PosixHostMatcher extends HostMatcher {
```

Comment:
In windows host names are case insensitive. There are few community members running on windows.